index.html

```diff
@@ -248,19 +248,14 @@ <h2 class="heading">JHipster news</h2>
                         <ul>
-                            <li>September 9th, 2020: at <a href="https://www.jug-mz.de/" target="_blank" rel="noopener">JUG Mainz</a>, <a href="https://twitter.com/atomfrede" target="_blank" rel="noopener">Frederik Hahne</a> will do a session about rapid application development with JHipster.<a href="https://www.meetup.com/JUG-Mainz/events/272558268/" target="_blank" rel="noopener">RSVP on Meetup.com</a>.</li>
-                            <li>September 26th, 2020: at <a href="https://www.meetup.com/JHipster-Nigeria">JHipster Nigeria User Group</a>, <a href="https://twitter.com/pibesson" target="_blank" rel="noopener">Pierre Besson</a> and <a href="https://twitter.com/deepu105" target="_blank" rel="noopener">Deepu K Sasidharan</a> will do a session on Building Applications With Microservices and Docker Using JHipster. <a href="https://www.meetup.com/JHipster-Nigeria/events/271992097/" target="_blank" rel="noopener">RSVP on Meetup.com</a>.</li>
-                            <li>October 29th, 2020: at <a href="https://www.jcon.one/" target="_blank" rel="noopener">JCON</a>, <a href="https://twitter.com/atomfrede" target="_blank" rel="noopener">Frederik Hahne</a> will do a session about 'Whats new in JHipster 7'.</li>
+                            <li>February 9th, 2021: at <a href="https://java.geekle.us/" target="_blank" rel="noopener">Geekle Java Summit</a>, <a href="https://twitter.com/deepu105" target="_blank" rel="noopener">Deepu K Sasidharan</a> will do a session about <b>Building microservices with JHipster</b>.</li>
                         </ul>
                         <h3>News</h3>
                         <ul>
-			    <li><b>Full Stack Development with JHipster - Second edition</b> by Deepu K Sasidharan and Sendil Kumar is published. Get it on <a href="https://www.packtpub.com/web-development/full-stack-development-with-jhipster-second-edition" target="_blank" rel="noopener">Packt</a> and <a href="https://smile.amazon.com/Full-Stack-Development-JHipster-microservices/dp/1838824987" target="_blank" rel="noopener">Amazon</a>. Get 25% discount on E-books from Amazon and Packt using discount code "25JAVASILVER". Valid until 1st of July 2020</li>
+                            <li><b>The JHipster Developers Association</b> now hosts <b>Virtual Meetups</b> on its <a href="https://www.youtube.com/c/JHipsterDevelopersAssociation">Youtube Channel</a>.</li>
+			                <li><b>Full Stack Development with JHipster - Second edition</b> by Deepu K Sasidharan and Sendil Kumar is published. Get it on <a href="https://www.packtpub.com/web-development/full-stack-development-with-jhipster-second-edition" target="_blank" rel="noopener">Packt</a> and <a href="https://smile.amazon.com/Full-Stack-Development-JHipster-microservices/dp/1838824987" target="_blank" rel="noopener">Amazon</a>. Get 25% discount on E-books from Amazon and Packt using discount code "25JAVASILVER". Valid until 1st of July 2020</li>
```

Comment:
indent?

Reply:
I personally don't see the indent problem.